src/coord/src/catalog/builtin.rs

```diff
@@ -1895,12 +1896,17 @@ mod tests {
         });
 
         let rows = client
-            .query("SELECT oid, proname, proargtypes FROM pg_proc", &[])
+            .query(
+                "SELECT oid, proname, proargtypes, prorettype, proretset FROM pg_proc",
+                &[],
+            )
             .await?;
 
         struct PgProc {
             name: String,
             arg_oids: Vec<u32>,
+            ret_oid: Option<u32>,
+            ret_set: bool,
```

Comment:
Don't we need a migration here as well? Or since it's not persisted just changing the schema here is fine?